app.go

```diff
@@ -11,9 +11,12 @@ import (
 	"io/ioutil"
 	"net/http"
 	"net/url"
+	"strconv"
 	"strings"
 	"time"
 
+	"github.com/cezarsa/form"
```

Comment:
I've been wondering for a while, how secure is it to use github sources? are they saved in local to some extent?